src/Elastic.Apm.AspNetCore/ApmMiddlewareExtension.cs

```diff
@@ -58,10 +56,7 @@ public static class ApmMiddlewareExtension
 		)
 		{
 			var subs = new List<IDiagnosticsSubscriber>(subscribers ?? Array.Empty<IDiagnosticsSubscriber>());
-			subs.Add(new AspNetCoreDiagnosticsSubscriber());

```

Comment:
This one should not be removed. - that's how we capture exceptions within the request.

Reply:
👍